addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -48,10 +48,27 @@ def send_mail_test(self):
                 'notification': True,
                 'mailing_id': mailing.id,
                 'attachment_ids': [(4, attachment.id) for attachment in mailing.attachment_ids],
-                'auto_delete': True,
```

Comment:
I would give auto_delete to False explicitly (otherwise I might add it in some random "wtf are people doing with their emails" task)

Reply:
Haha yes I can see that happening, I added an explicit False.